lib/lib_rss.php

```diff
@@ -578,3 +578,21 @@ function validateShortcutList($shortcuts) {
 	}
 	return $shortcuts_ok;
 }
+
+function errorMessage($errorTitle, $error = '') {
+	// Prevent empty <h2> tags by checking if error isn't empty first
+	if ('' !== $error) {
+		$error = "<h2>{$error}</h2>";
+	}
+	return <<<MSG
+	<h1>{$errorTitle}</h1>
+	{$error}
+	<pre>
```

Comment:
This is making a lot of assumptions about the user's configuration, OS, namings, and I am afraid it might often-times be more confusing than helpful. I will make a few suggestions